grails-app/conf/Config.groovy

```diff
@@ -883,6 +897,15 @@ openboxes {
                     }
                 }
             }
+            numberOfOpenPurchaseOrders {
+                enabled = true
+                title = "react.dashboard.numberOfOpenPurchaseOrders.title.label"
+                info = "react.dashboard.numberOfOpenPurchaseOrders.info.label"
+                subtitle = "react.dashboard.subtitle.purchaseOrders.label"
+                numberType = 'number'
+                type = 'number'
+                endpoint = "/${appName}/api/dashboard/openPurchaseOrdersCount"
```

Comment:
Maybe _open**ed**PurchaseOrdersCount_ will be better?

Reply:
I think `open` is fine (same wording was used in the ticket)